src/router/Router.tsx

```diff
@@ -7,9 +7,10 @@ import Layout from '~elements/Layout'
 import LayoutAuth from '~elements/LayoutAuth'
 import LayoutProcessCreate from '~elements/LayoutProcessCreate'
 import { StripeCheckout, StripeReturn } from '~elements/Stripe'
-import OrganizationProtectedRoute from './OrganizationProtectedRoute'
+import { default as ScaffoldOrganizationProtectedRoute } from './OrganizationProtectedRoute'
 
 import { SuspenseLoader } from './SuspenseLoader'
+import SaasOrganizationProtectedRoute from '~src/router/SaasOrganizationProtectedRoute'
```

Comment:
@selankon The import should be from `'./SaasOrganizationProtectedRoute'`

Reply:
ef732d4